apps/src/storage/firebaseUtils.js

```diff
@@ -96,6 +99,18 @@ export function getProjectDatabase() {
   return getFirebase(config.firebaseName).child(path);
 }
 
+export function getPathRef(dbRef, path) {
+  return dbRef.child(escapeFirebaseKey(path));
+}
```

Comment:
Nice extraction! 

I think it would be helpful to add a comment on this method explaining when to use this vs calling .child() directly or using another helper. I see we don't use this method for accessing config paths. So, is this needed to access anything within a specific project? Also, is dbRef necessarily the project's root node? If accurate, names like getProjectPathRef and projectDbRef could make it easier to understand and may reduce the amount of comments needed.